src/Core/Resolvers/Sql Query Structures/BaseSqlQueryStructure.cs

```diff
@@ -361,8 +360,8 @@ protected static object ParseParamAsSystemType(string param, Type systemType)
                 "Double" => double.Parse(param),
                 "Decimal" => decimal.Parse(param),
                 "Boolean" => bool.Parse(param),
-                "DateTime" => DateTimeOffset.Parse(param, DateTimeFormatInfo.InvariantInfo, DateTimeStyles.AssumeUniversal),
-                "DateTimeOffset" => DateTimeOffset.Parse(param, DateTimeFormatInfo.InvariantInfo, DateTimeStyles.AssumeUniversal),
+                "DateTime" => DateTime.Parse(param),
+                "DateTimeOffset" => DateTimeOffset.Parse(param),
```

Comment:
This is not required when feeding the values to the database. Hotchocolate converts date/time to UTC while returning the parsed result but not for values that are feeded for parameters.

Reply:
I added it because when inserting Hotchocolate was converting it to local timezone.
![image](https://github.com/Azure/data-api-builder/assets/102276754/65a05fc7-f8ef-471f-83a2-eb4865fe62a8)

can you check if after your changes something like this is not happening. Even if it's not happening, i feel it's good to have it explicitly in the code for better clarity that we want it to assume it's in UTC and not to apply any conversion.

Reply:
Confirmed. Not happening.